CMakeLists.txt

```diff
@@ -161,9 +162,17 @@ if(ROBOTOLOGY_ENABLE_DYNAMICS)
   find_or_build_package(walking-controllers)
   find_or_build_package(whole-body-estimators)
   find_or_build_package(whole-body-controllers)
+  find_or_build_package(bipedal-locomotion-framework)
```

Comment:
The name here should match the one of the `Build<package>.cmake`  and the one passed to `ycm_ep_helper` . Even if not strictly necessary, it is a good idea to just use the cmake package name (so in this case `BipedalLocomotionFramework`), because `find_or_build_package` mirrors `find_package`, so if we have: 
~~~
find_package(BipedalLocomotionFramework)
~~~
we should also have: 
~~~
find_or_build_package(BipedalLocomotionFramework)
~~~